### PR TITLE
Add ACI signing documentation

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: quay-enterprise/aci-signing-keys.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Generating signing keys for ACI conversion
+
```
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: "This document explains how to add a pair of signing keys to your Quay Enterprise installation allows Quay to sign container images after converting them to ACI format."

File: quay-enterprise/aci-signing-keys.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Download the generation script and config
+
+Download the files [aci-signing-key-batch](aci-signing-key-batch) and [generate-signing-keys.sh](generate-signing-keys.sh) next to your configuration directory.
```
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: Download the configuration file [`aci-signing-key-batch`]() and the [`generate-signing-keys.sh`]() script to your configuration directory.

File: quay-enterprise/aci-signing-keys.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Make sure the generation script is executable
```
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: Make `generate-signing-keys.sh` executable:

(not a heading)

File: quay-enterprise/aci-signing-keys.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+```sh
+chmod +x generate-signing-keys.sh
+```
+
+## Edit the configuration
+
+Edit the [aci-signing-key-batch](aci-signing-key-batch) configuration, replacing the email address, name and comment.
```
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: s/name/name,/
s/comment/comment with values appropriate for your site/

File: quay-enterprise/aci-signing-keys.md (hunk 1 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: Added

File: quay-enterprise/aci-signing-keys.md (hunk 3 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: "Not a heading"? Just text?

File: quay-enterprise/aci-signing-keys.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Run the generation script
+
+Run the generation script, pointing it to an output config directory.
```
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: Run the generate script, giving the name of an output directory as the argument:

File: quay-enterprise/aci-signing-keys.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+```sh
+./generate-signing-keys.sh outputdir
+```
+
+The files will be generated as `signing-private.gpg` and `signing-public.gpg`:
```
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: The script will create a pair of files beneath the given directory named `signing-private.gpg` and `signing-public.gpg`.

File: quay-enterprise/aci-signing-keys.md (hunk 4 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: Dat Oxford Comma doh

File: quay-enterprise/aci-signing-keys.md (hunk 5 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: Done

File: quay-enterprise/aci-signing-keys.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+gpg: done
+Generating public signing key
+Determining private key
+Exporting private signing key
+Private key name: CBFB447F
+Cleaning up
+Emitted outputdir/signing-private.gpg and outputdir/signing-public.gpg
+```
+
+Take note of the private key's name.
```
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: Isn't the private key's name always "signing-private.gpg"?

File: quay-enterprise/aci-signing-keys.md (hunk 6 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: Done

File: quay-enterprise/aci-signing-keys.md (hunk 7 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: Name is different from filename.

File: quay-enterprise/aci-signing-keys.md (hunk 3 above)
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: yeah, download/make exec. are one step, don't need two headings.

File: quay-enterprise/aci-signing-keys.md (hunk 7 above)
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: Aha - d'oh.

File: quay-enterprise/aci-signing-keys.md (hunk 3 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: ok

File: quay-enterprise/aci-signing-keys.md (hunk 7 above)
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: Take note of the *name* generated for the private key. It is `CBFB447F` in this example.

File: quay-enterprise/aci-signing-keys.md (hunk 3 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: Done

File: quay-enterprise/aci-signing-keys.md (hunk 7 above)
Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: Let me add some clarification

File: quay-enterprise/aci-signing-keys.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+Private key name: CBFB447F
+Cleaning up
+Emitted outputdir/signing-private.gpg and outputdir/signing-public.gpg
+```
+
+Take note of the private key's name (example from above: `CBFB447F`)
+
+## Enter config in superuser panel
+
+Upload the `signing-private.gpg` and `signing-public.gpg` files as the keys, and enter the generated key's name. Save the configuration and restart Quay Enterprise to enable signed ACI conversion.
```
Comment (joshix) [inline quay-enterprise/aci-signing-keys.md]: Visit the Quay superuser panel. Upload the pair of key files, and enter the generated private key name. Save the configuration and restart Quay Enterprise to enable signing of converted ACIs.

Comment (josephschorr) [inline quay-enterprise/aci-signing-keys.md]: Done

**Comment by joshix** [comment]
lgtm

**Pull request merged**